docs/whats-new/dotnet-8.md

```diff
@@ -175,6 +176,8 @@ Explicit package references should also be added to your *.csproj* file for the
 </ItemGroup>
 ```
 
+When using [Central Package Management (CPM)](/nuget/consume-packages/central-package-management), MAUI packages are no longer referenced automatically and must be explicitly added. For projects that only contain assets (`MauiImage`, `MauiAsset`, etc.), it is sufficient to add the `Microsoft.Maui.Resizetizer` package (instead of `Microsoft.Maui.Controls`).
```

Comment:
```suggestion
When using [Central Package Management (CPM)](/nuget/consume-packages/central-package-management), .NET MAUI packages are no longer referenced automatically and must be explicitly added. For projects that only contain assets (`MauiImage`, `MauiAsset`, etc.), it is sufficient to add the `Microsoft.Maui.Resizetizer` package (instead of `Microsoft.Maui.Controls`).
```